src/transport/PeerAddress.h

```diff
@@ -48,7 +48,7 @@ enum class Type
     kUndefined,
     kUdp,
     kBle,
-    // More constants to be added later, such as TCP
```

Comment:
Should we maintain this comment or is TCP the last constant we plan to add?

Reply:
hmmm ... I had removed this in some intermediate versions. Should be fixed now.

Reply:
I am not aware of anything else we could add as chip spec. I believe a comment would be 'add more stuff here if needed' however that seems redundant as any enum would be 'add more if needed'.